src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rand = new Random();
+
     public String getRandomColor() {
-        return null;
+
+        return Colors.values()[rand.nextInt(Colors.values().length)].toString();
```

Comment:
Instead of using `toString()`, it's recommended to use `name()` for getting the String representation of enum constants to ensure you get the exact name of the enum constant.